zanata-adapter-properties/src/main/java/org/zanata/adapter/properties/PropWriter.java

```diff
@@ -43,6 +43,30 @@ public static void writeUTF8(final Resource doc, final File baseDir)
         write(doc, baseDir, true);
     }
 
+    /**
+     * Writes to given properties file of the given TranslationsResource
+     * @param doc
+     * @param propertiesFile
+     * @throws IOException
+     */
+    public static void write(final Resource srcDoc,
```

Comment:
Is it possible to call this `writeTranslationsLatinOne` so it is much quicker to tell what it does compared to the other `write` methods?